process-engine/bpmn-studio
# ♻️ Improve performance of inspect/dashboard

## Changes

The first step to improve the performance of the inspect/dashboard view
1. Refactor cronjob-list
2. Refactor process-list
3. Refactor task-list
4. Remove unnecessary claim checks from the dashboard

## Issues

Part of #1594 

PR: #1733 

## How to test the changes

- create a large database
- navigate to inspect
- see that there is no white window for some seconds
